main.nf

```diff
@@ -804,7 +808,26 @@ if (params.saturation) {
 
 
 /*
- * STEP 10 Post peak calling processing
+ * STEP 10.1 Prepare annotation file
+ */
+
+ process prepare_anno {
+
+     output:
+     file '*.{gtf}' into annotation
+
+     when: gtf
+
+     script:
+     """
+     awk -F '\\t' '{print \$9}' $gtf | sed -e 's/^.*gene_id //g' | awk -F";| " '{print \$1 "\\t" \$4}' > genes.txt
+     awk -F '\\t' '{print \$1 "\\t" \$4 "\\t" \$5 "\\t" \$7}' $gtf > coordinates.txt
```

Comment:
We should stage the GTF file as an `input`. It sometimes works without doing this (as I'm sure it did when you tested it), but it can break without it in some circumstances as Nextflow does clever Docker / Singularity directory mounting and stuff if it's an input channel.

Reply:
Fixed